### DISCUSSION TITLE
Data importer manual execution processing

### DISCUSSION THREAD

**Discussion by SergeZ23** [pimcore/pimcore#10082] [Q&A / Support] [answered]
Hi, my server only allows cron tasks to run every 5 minutes and no more frequently.
I was wondering if it's possible to not only load import entries from the source, but also process them on click of manual execution "Start" button.
I'm not a professional programmer and don't have experience with php, so maybe there's a somewhat straight forward method to achieve this?

**Comment by fashxp** [comment, marked as answer]
on short notice, you could use the [Process Manager](https://github.com/elements-at/ProcessManager), configure the import command there (probably need to [white list](https://github.com/elements-at/ProcessManager/blob/master/doc/commandsValidator.md) it) and then execute the import manually via the process manager.

**Reply by SergeZ23** [reply]
Thanks for the suggestion. I had a quick look at Process Manager and it seems that it also executes commands via Cronjob

> * * * * * php ~/www/bin/console process-manager:maintenance > /dev/null 2>&1

Which kinda defeats the purpose, as I can only add Cronejob as */5 * * * *
Or am I wrong and command can be executed without Cron using this bundle?

**Reply by fashxp** [reply]
you should be able to run them manually too

**Reply by SergeZ23** [reply]
Thank you, I'll try it.
